.travis.yml

```diff
@@ -0,0 +1,25 @@
+# see https://docs.travis-ci.com/user/reference/trusty/#container-based-with-sudo-false
+sudo: false
+
+# see https://docs.travis-ci.com/user/languages/cpp/
+language: cpp
+
+# For build environment setup
+# see https://docs.travis-ci.com/user/reference/overview/
+matrix:
+  include:
+    # see https://docs.travis-ci.com/user/languages/cpp/#gcc-on-linux
+    - os: linux
+      dist: trusty
+      compiler: gcc
+
+    # see https://docs.travis-ci.com/user/languages/cpp/#clang
+    - os: osx
+      osx_image: xcode9.4
+      compiler: clang
+
+script:
+  - cmake --version
+  - cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DABSL_RUN_TESTS=ON -DABSL_USE_GOOGLETEST_HEAD=ON
```

Comment:
All else being equal, I'd rather choose Debug over Release, but that's unfortunate we can't do better.  Can we not set a BUILD_TYPE env var in the matrix so we get better config coverage?